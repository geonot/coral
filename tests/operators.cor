fn log(message: string):
    return

log('10 + 3 = {10 + 3}')
log('10 - 3 = {10 - 3}')
log('10 * 3 = {10 * 3}')
log('10 > 3 is {10 > 3}')
log('10 < 3 is {10 < 3}')
log('10 >= 10 is {10 >= 10}')
log('10 <= 10 is {10 <= 10}')
log('10 == 10 is {10 == 10}')
log('10 != 3 is {10 != 3}')
log('true and false is {true and false}')
log('true or false is {true or false}')
log('not true is {!true}')
log('neg 10 is {-10}')
