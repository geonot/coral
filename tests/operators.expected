10 + 3 = 13
10 - 3 = 7
10 * 3 = 30
10 > 3 is true
10 < 3 is false
10 >= 10 is true
10 <= 10 is true
10 == 10 is true
10 != 3 is true
true and false is false
true or false is true
not true is false
neg 10 is -10
